widgetssdk/src/main/java/com/glia/widgets/launcher/ConfigurationManager.kt

```diff
@@ -14,17 +14,26 @@ internal interface ConfigurationManager {
 
     val queueIdsObservable: Flowable<List<String>>
 
+    val visitorContextAssetId: String?
+
     /**
      * Applies [GliaWidgetsConfig] from the [GliaWidgets.init] configuration step
      */
     fun applyConfiguration(config: GliaWidgetsConfig)
 
     /**
-     * Retrives queue IDs
+     * Sets queue IDs
      * [queueIds] A list of queue IDs to be used for the engagement launcher.
      * When empty or invalid, the default queues will be used.
      */
     fun setQueueIds(queueIds: List<String>)
+
+    /**
+     * Sets the visitor context for the engagement.
+     *
+     * @param visitorContextAssetId a visitor context id from Glia Hub.
+     */
```

Comment:
This is an internal class and the method name is self explanatory. IMHO think no javadoc is needed here

Reply:
I see this in many other places. It is not a problem but I just don't see the point.

Reply:
You are right. It's for our internal use. I just saw that other functions are documented in the `ConfigurationManager` interface and decided to follow this approach. But I agree with you.